iris-mpc-utils/src/aws/config.rs

```diff
@@ -74,6 +77,9 @@ impl AwsClientConfig {
         sqs_response_queue_url: String,
         sqs_wait_time_seconds: usize,
     ) -> Self {
+        let config = get_sdk_config().await;
+        if config.region().is_none() {}
```

Comment:
What's happening here? Is `config` used? I see `get_sdk_config` is called again a few lines below.